packages/bff/src/graphql/types/mutation.ts

```diff
@@ -111,3 +111,25 @@ export const DeleteFavoriteActor = extendType({
     });
   },
 });
+
+export const UpdateLanguage = extendType({
+  type: 'Mutation',
+  definition(t) {
+    t.field('updateLanguage', {
+      type: Response,
+      args: {
+        language: stringArg(),
+      },
+      resolve: async (_, { language }, ctx) => {
+        try {
+          const pid = ctx.session.get('pid');
+          await updateLanguage(pid, language);
+          return { success: true, message: `Success, language updated to ${language}` };
```

Comment:
Not sure if need this `message` field, to be honest, since we are not using it  and it's just more bytes over the wire.